Library/OcMainLib/OpenCorePlatform.c

```diff
@@ -951,6 +951,7 @@ OcGetLegacySecureBootECID (
   }
 
   CopyMem (ApECID, &Uuid, sizeof (*ApECID));
+  DEBUG ((DEBUG_INFO, "OC: ApECID %ld from system-id\n", *ApECID));
```

Comment:
if I remember correctly, it should be %Ld for a 64-bit number. I am also not positive whether it should be her though.

Reply:
I'm not sure there is any other way to tell what's finally used? (When I wanted to know this, for sure, at some point, I didn't think I could otherwise tell?)

Reply:
Fair. Just change the print format then.